lib/src/kvs/rocksdb/mod.rs

```diff
@@ -282,12 +288,12 @@ impl Transaction {
 		let inner = self.inner.lock().await;
 		let inner = inner.as_ref().unwrap();
 		// Get the arguments
-		let key = key.into();
+		let key_rocksdb: Key = key.into();
```

Comment:
The naming of key_rocksdb is inconsistent, the other implementations just use key